src/components/styled/DashedOutline.ts

```diff
@@ -0,0 +1,10 @@
+import { styled } from "@mui/material/styles";
+import { Box } from "@mui/material";
+import theme from "theme";
+
+const DashedOutline = styled(Box)`
```

Comment:
Shouldn't this (and the ErrorMessage.ts and SolidOutline.ts) go under elements/styled instead of components/styled?